ocw/lib/EC2.py

```diff
@@ -59,6 +61,21 @@ def ec2_client(self, region='eu-central-1'):
                                                      region_name=region)
         return self.__ec2_client[region]
 
+    def eks_client(self, region='eu-central-1'):
+        if region not in self.__eks_client:
+            self.__eks_client[region] = boto3.client('eks', aws_access_key_id=self.__key,
+                                                     aws_secret_access_key=self.__secret,
+                                                     region_name=region)
+        return self.__eks_client[region]
+
+    def list_clusters(self):
```

Comment:
If you take a look to `list_instances()` we only give the vm's of one region.
I somehow would prefer to have some kind of equal behavior in the methods.
So maybe we add a `_all_regions` to the method name then it is clear to me, that it behaves differently then `list_xxxx()`.
WDYT?